docs/customer-360/customer-journey/index.md

```diff
@@ -0,0 +1,57 @@
+---
+seo:
+  title: Track a Customer Journey
+  description: This recipe demonstrates how to use ksqlDB to track a customer journey through web pages online.
+---
+
+# Track a Customer Journey
+
+Companies with online websites, and customers that browse those websites, want to know which web pages their customers have visited.
```

Comment:
> online websites

Are not all websites online?

Reply:
@ybyzek @MichaelDrogalis this recipe seems very similar to the "Understand user behavior with clickstream data." Is the difference that this one is just looking at what pages were visited and how long someone stayed ont he page? VS. Clickstream is looking at what a customer journey might be through the web. 

Could you share with me which customer or how this use case was used. To second @mjsax's question, I'm not 100% clear why an org would just want to know what pages an individual customer visited. I could potentially see the use case for seeing which webpage is most popular.

Reply:
> @ybyzek @MichaelDrogalis this recipe seems very similar to the "Understand user behavior with clickstream data." Is the difference that this one is just looking at what pages were visited and how long someone stayed ont he page? VS. Clickstream is looking at what a customer journey might be through the web.
> 
> Could you share with me which customer or how this use case was used. To second @mjsax's question, I'm not 100% clear why an org would just want to know what pages an individual customer visited. I could potentially see the use case for seeing which webpage is most popular.

FWIW this has come up a few times now, a few comments from the associated issue: https://github.com/confluentinc/ksqldb-recipes/issues/106#issuecomment-1005933605

Reply:
The queries are different and I think we should work to present them separately as well.  e.g. clickstream could focus on metadata like number of errors per minute and pages per minute, whereas customer journey could focus on individual web pages visited.